From e0ceac7c48a4cc616f1602d4cc3430a306740021 Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Mon, 13 Jul 2015 13:12:12 -0400
Subject: [PATCH 16/20] Ticket 47878 - Remove warning suppression in 1.3.4

Description:  Perl 5.16 does not support the warning suppression of
              "smartmatch", but 5.18 does.  RHEL 7 is currently only
              using 5.16, so it needs to be removed from 1.3.4 for now.

https://fedorahosted.org/389/ticket/47878

Reviewed by: nhosoi(Thanks!)

(cherry picked from commit d8b7a3c93692eae229c74ef67f6f608f95eb2eef)
---
 ldap/admin/src/scripts/52updateAESplugin.pl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ldap/admin/src/scripts/52updateAESplugin.pl b/ldap/admin/src/scripts/52updateAESplugin.pl
index ae258b6..4225770 100644
--- a/ldap/admin/src/scripts/52updateAESplugin.pl
+++ b/ldap/admin/src/scripts/52updateAESplugin.pl
@@ -6,7 +6,7 @@ use File::Basename;
 use File::Copy;
 use DSUtil qw(debug serverIsRunning);
 
-no warnings 'experimental::smartmatch';
+# no warnings 'experimental::smartmatch';   warning supression available in perl 5.18
 
 #
 # Check if there is a DES plugin and make sure the AES plugin contains the same attributes
-- 
1.9.3

